ts-sdk/whirlpool/src/config.ts

```diff
@@ -49,6 +58,18 @@ export async function setWhirlpoolsConfig(
     );
 }
 
+/**
+ * Sets the WhirlpoolsConfig address based on the selected network.
+ *
+ * @param {keyof typeof WHIRLPOOLS_CONFIG_ADDRESSES} network - The network to set. Valid options are: SolanaMainnet, SolanaDevnet, EclipseMainnet, EclipseTestnet.
+ * @throws {Error} If the network is invalid.
+ */
+export async function setNetwork(
```

Comment:
Nit: this is not actually setting the network so not sure if the naming is fully appropriate. You could overload the already existing 'setWhirlpoolsConfig' function.

The solana cli for example takes a rpcUrlOrCanonialName parameter when setting the rpc url

Reply:
You are right. I went for this name, because I wanted to approach it from the perspective of the developer with the goal to abstract away the notion of WhirlpoolsConfig account.

How about `configureNetwork`, `setNetworkConfig`, or `useNetwork`?

If you prefer the overload option, I would just have to explain it well in the Docs.

Reply:
> How about configureNetwork, setNetworkConfig, or useNetwork?

Think all of these still have the same problem. This function is not doing anything with the network. It is setting the whirlpoolConfigAddress based on the network. I guess maybe something like `setWhirlpoolsConfigByNetwork` could work but then we might as well overload it. Another option could be `configure` or `configureSdk` but not sure if that is descriptive enough either